src/components/EggheadPlayer/players/Bitmovin.js

```diff
@@ -329,6 +339,11 @@ export default class Bitmovin extends Base {
     this.player && this.player.pause()
   }
 
+  isFullscreen() {
```

Comment:
this is a façade over the bitmovin player so we need to wrap underlying methods

fun fact, you forget this every couple of years